src/api/app/controllers/webui/patchinfo_controller.rb

```diff
@@ -98,6 +98,11 @@ def save
         @file.each(:package) do |pkg|
           node.package pkg.text
         end
+        @file.each(:releasetarget) do |release_target|
```

Comment:
the `save` method is huge! :scream: maybe it would be a good idea to think about how to get code out of here instead of adding more code :thinking:

Reply:
@Ana06:

> the `save` method is huge! :scream: maybe it would be a good idea to think about how to get code out of here instead of adding more code :thinking:

I created an issue for refactoring this method: #5081